oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/service/EventAnalysisService.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.receiver.event.service;
+
+import org.apache.skywalking.apm.network.event.v3.Event;
+import org.apache.skywalking.oap.server.library.module.Service;
+
+/**
+ * Analyze the collected event data.
+ */
+public interface EventAnalysisService extends Service {
+    void analyze(Event event);
+}
```

Comment:
Why do we need this as a `Service`? `Service` is the internal concept for across-module interaction, as a receiver and especially event receiver, I don't think this is the case.
And if this will(for event, may not), you could find there are `*-analyzer`(s) module exposing the interface.